servers/apps/listmonk/README.md

```diff
@@ -0,0 +1,21 @@
+# Listmonk 
+
+[Listmonk](https://listmonk.app) is a bulk email sending service which is used heavily at FOSS United for sending out bulk emails to attendees, speakers and in general to the whole FOSS Community.
+
+Server: Listmonk (Nanode: 1GB) 
+App: Listmonk
+Latest updated version: ```v3.0.0 (f9120d9 2024-02-04T11:20:27Z, linux/amd64)```
+Install Dir: `/opt/listmonk`
+Backups Dir: `/opt/backups`
+OS: Ubuntu 20.04 LTS
+RAM: 1Gib
```

Comment:
RAM is 1GB not Gib(Gibibyte)?

Reply:
Ohh cool, will update it. It seems, I was wrong all this time about GiB being GigaByte. As `cfdisk` command in linux calculates Gigabytes as GiB. 

Dumb thing, but yeah fine.